init.lua

```diff
@@ -4,17 +4,28 @@ local json = require('json')
 local fiber = require('fiber')
 local log = require('log')
 
+
 box.cfg({
     memtx_dir='db',
     wal_dir='db'
 })
 box.once('migration', function()
+    if type(box.space.jobs) == 'table' then
+        box.space.jobs:drop()
+    end
+
    jobs = box.schema.space.create('jobs')
-   jobs:format({{name = 'id', type = 'unsigned'},
+   jobs:format({{name = 'run_id', type = 'unsigned'},
+                {name = 'run_attempt', type = 'unsigned'},
+                {name = 'run_status', type = 'unsigned'},
```

Comment:
`run_status`, `run_conclusion` and `job_conclusion` seem to be strings. https://www.tarantool.io/en/doc/latest/concepts/data_model/value_store/#data-types